src/app/app.cpp

```diff
@@ -1832,7 +1832,12 @@ void CApplication::SetLanguage(Language language)
     // Update C++ locale
     try
     {
+#if defined(_MSC_VER) && defined(_DEBUG)
+        // Avoids failed assertion in VS debugger
+        throw -1;
```

Comment:
Did you check that translations work correctly with this code?

Reply:
No, I did not :(

Reply:
Nothing seems broken, the game starts in Polish on my system. And even if translations were broken, I don't think it's really important for debugging.